services/requester/requester.go

```diff
@@ -326,11 +326,16 @@ func (e *EVM) Call(
 	if err != nil {
 		return nil, fmt.Errorf("failed to decode EVM result from call: %w", err)
 	}
+
 	if evmResult.ErrorCode != 0 {
-		return nil, getErrorForCode(evmResult.ErrorCode)
+		if evmResult.ErrorCode == evmTypes.ExecutionErrCodeExecutionReverted {
+			return nil, errs.NewRevertError(evmResult.ReturnedData)
+		} else {
```

Comment:
no need for `else`

Reply:
Oh, right :sweat_smile: 
Removed in https://github.com/onflow/flow-evm-gateway/pull/297/commits/90d72ce4383b1d74312de9d2f8074af0a9283ca6